src/test/java/screens/GraphSceneTest.java

```diff
@@ -17,11 +18,14 @@
 import org.mockito.runners.MockitoJUnitRunner;
 
 import javax.swing.*;
+import java.lang.reflect.Method;
 import java.util.ArrayList;
+import java.util.LinkedList;
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.TimeUnit;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
 import static org.mockito.Mockito.*;
```

Comment:
Please do not use * imports. If intellij replaces a lot of imports from the same package/class to * imports, you should go to file -> settings -> editor -> code style -> java and go to the tab imports. You can set the number of imports necessary to replace them with * import.